docs/zopebook/InstallingZope.rst

```diff
@@ -279,7 +279,7 @@ Zope instance fails to start, make sure that another application isn't
 already running on the same TCP port (8080).
 
 Zope also has the capability to listen on other TCP ports.  Zope supports
-separate TCP ports for FTP (File Transfer Protocol), "monitor" (internal
+separate TCP ports for "monitor" (internal
```

Comment:
I don't think that monitor port still exists. I think that was handled by ZServer.